modules/monitoring-choosing-a-metrics-collection-profile.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="choosing-a-metrics-collection-profile_{context}"]
+= Choosing a metrics collection profile
+
+To choose a metrics collection profile for core {product-title} monitoring components, edit the `cluster-monitoring-config` `ConfigMap` object.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have enabled Technology Preview features by using the `FeatureGate` custom resource (CR).
```

Comment:
we don't need to enable Technology Preview features by using `FeatureGate`, there is scrapeProfile parameter for prometheusK8s, sorry for misleading in my previous comment. leave it to @JoaoBraveCoding help confirm

Reply:
I had forgotten to add this but in my latest commit to the PR I added this, so this is indeed correct we need to enable TechPreview